### DISCUSSION TITLE
Kasowanie null dla "okienka".

### DISCUSSION THREAD

**Discussion by tiwek** [Czapla-dev/vulcan-for-hassio#24] [Ideas]
Na początek: Świetna robota.
Ale (zawsze jest jakieś ale), można by pomyśleć o jakimś innym wyświetlaniu dla okienek. W tej chwili mam tak:
![image](https://user-images.githubusercontent.com/74269691/156352251-992a889b-c57d-40b0-bc0f-98efad3326fd.png)

Może tego nulla po prostu kasować i zostawiać puste ?
Bo godziny na pewno powinny zostać.

**Comment by Antoni-Czaplicki** [comment]
Chyba wystarczy dodać [tutaj](https://github.com/Antoni-Czaplicki/vulcan-for-hassio/blob/4b06f135975af25bcabff491d305f0e7cbcdd688/custom_components/vulcan/calendar.py#L161) coś w stylu `if event ["lesson"] is not null else "okienko"`, ale nie mam teraz czasu aby tym się zająć, jak chcesz możesz sam to zrobić i przetestować, a jak będzie działać to zapraszam do otwarcia PR.
